miniapp/build.gradle

```diff
@@ -105,6 +108,7 @@ dependencies {
 apply from: "../config/quality/checkstyle/android.gradle"
 apply from: '../config/quality/detekt/android.gradle'
 dependencies {
+    testImplementation project(path: ':miniapp')
```

Comment:
What is the need for this change? 🤔

Reply:
It's required without it the dependencies are failing for `MiniAppSecureStorageSpec`